hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkTableServiceClient.java

```diff
@@ -137,6 +137,7 @@ protected void completeClustering(
 
       LOG.info("Committing Clustering {} finished with result {}.", clusteringCommitTime, metadata);
       table.getActiveTimeline().transitionReplaceInflightToComplete(
+          false,
```

Comment:
why do we skip locking here completely.

Reply:
I guess it's because the file slicing is not affected by the replace operation, there is no need to keep atomicity.